rest/admin_api_test.go

```diff
@@ -290,24 +291,59 @@ function(doc, oldDoc) {
 	input = input + `]}`
 	response = rt.SendAdminRequest("POST", "/db/_bulk_docs", input)
 
+
 	// Start changes feed
 	var wg sync.WaitGroup
+
+	wg.Add(1)
+
+	numExpectedChanges := 201
+
 	go func() {
 		defer wg.Done()
-		wg.Add(1)
-		// Timeout allows us to read continuous changes after processing is complete.  Needs to be long enough to
-		// ensure it doesn't terminate before the first change is sent.
-		changesResponse := rt.Send(requestByUser("GET", "/db/_changes?feed=continuous&since=0&timeout=2000", "", "bernard"))
 
-		changes, err := readContinuousChanges(changesResponse)
-		assert.Equals(t, err, nil)
+		var since uint64
+		since = 0
+
+		maxTries := 10
+		numTries := 0
+
+		changesAccumulated := []db.ChangeEntry{}
+
+		for {
+
+			// Timeout allows us to read continuous changes after processing is complete.  Needs to be long enough to
+			// ensure it doesn't terminate before the first change is sent.
+			changesResponse := rt.Send(requestByUser("GET", fmt.Sprintf("/db/_changes?feed=continuous&since=%d&timeout=5000", since), "", "bernard"))
```

Comment:
If we're doing retry, do we actually want to bump the timeout from 2000 to 5000?

Reply:
Yeah, with the retry logic, this shouldn't matter.  I guess I was just trying to minimize how often it has to do a retry, but then again that adds some cost since it will never go faster than 5s.  Changed back to 2s.